sdc/crypto/encrypter.py

```diff
@@ -2,19 +2,20 @@
 from sdc.crypto.jwt_helper import JWTHelper
 
 
-def encrypt(json, key_store, key_purpose):
+def encrypt(json, key_store, key_purpose, encryption_service=None):
     """This encrypts the supplied json and returns a jwe token.
 
     :param str json: The json to be encrypted.
     :param key_store: The key store.
     :param str key_purpose: Context for the key.
+    :param Optional[str] encryption_service: The name of the encryption service
```

Comment:
Something like...

> The name of the downstream service the JWT is being encrypted for

...might clarify the arg

Reply:
Ya, I think it does. Have updated.